test/cpp/jit/test_utils.cpp

```diff
@@ -93,6 +93,25 @@ std::shared_ptr<Graph> build_lstm() {
   return g;
 }
 
+std::shared_ptr<Graph> build_mobile_export_analysis_graph() {
+  const auto graph_string = R"IR(
+    graph(%0 : Tensor):
+      %1 : int = prim::Constant[value=1]()
+      %2 : int = prim::Constant[value=2]()
+      %20 : int = prim::Constant[value=0]()
+      %21 : int = prim::Constant[value=9223372036854775807]()
+      %22 : str = prim::Constant[value="value"]()
+      %3 : Tensor  = aten::slice(%0, %1, %20, %2, %1)
```

Comment:
Would be nice to put comments of schemas for aten::slice.Tensor and aten::slice.str

Reply:
sure